### DISCUSSION TITLE
How to modify the third-party JS's style through TailwindCSS?

### DISCUSSION THREAD

**Discussion by tsq456** [tailwindlabs/tailwindcss#5138] [Help] [answered]
I integrated a third-party comment system to my blog, but I can't modify the style of comment system through `tailwincss.config.js`,
so how can I modify the third-party JS's style through TailwindCSS?

And if I want to add dark-mode to my site, is there possible that tailwindcss's dark-mode can control the third-party JS's style?

**Comment by simonswiss** [comment]
Hey!

Does your JS third-party comment system come with its own classes? Do you have control over the markup? Are you able to share some code?

The answer really depends on what the "styling story" for your comment system looks like. You might need to write "regular CSS", which you can totally do with Tailwind CSS - and perhaps use the `@apply` directives to compose styles that match the rest of your system.

Really would need to see some code to be able to help.

**Comment by tsq456** [comment]
Thanks for your reply!

Yeah, the comment system has its own classes. And I found that I can control the style through adding my own CSS code in the tailwind.scss.

but I encountered another problem that how can I compile my own SCSS code in the tailwind.css to normal CSS, Because I found that the format of SCSS is still maintained after compiled to normal CSS .

You can see my code here https://github.com/tsq456/hugo-theme-windy/blob/45ec94e652c6ef6e8e66bcdf7fde018db7ae4e30/themes/hugo-theme-windy/assets/scss/tailwind.scss#L11

**Comment by simonswiss** [comment, marked as answer]
Hey!

By default, the CLI will only run your code through `tailwindcss` and `autoprefixer`, so it won't do anything about your SCSS syntax.

To compile your SCSS styles, you need to add a PostCSS plugin like this one:

https://www.npmjs.com/package/postcss-sass

You'll also need to add a `postcss.config.js` file at the root of your project, so the CLI knows which plugins to run your styles through:

https://tailwindcss.com/docs/installation#using-a-custom-post-css-configuration

You'll want to run the styles through the `postcss-sass` plugin first, and then Tailwind

Hopefully that gets you going!

**Reply by tsq456** [reply]
Thanks!

I added the PostCSS plugin and the `postcss.config.js` to my project, but it did not make any difference.

Is there something wrong with the configuration in my `package.json` file? 

``` 
 "scripts": {
    "dev": "cross-env NODE_ENV=development npx tailwindcss -i ./themes/hugo-theme-windy/assets/scss/tailwind.scss --postcss -o ./themes/hugo-theme-windy/assets/scss/main.css --jit -w",
    "build": "cross-env NODE_ENV=production npx tailwindcss -i ./themes/hugo-theme-windy/assets/scss/tailwind.scss --postcss -o ./themes/hugo-theme-windy/assets/scss/main.css --jit --minify"
  },
```

**Reply by tsq456** [reply]
It's Okay now, I follow the DOC below and it output the right code now.

https://tailwindcss.com/docs/using-with-preprocessors#using-sass-less-or-stylus

Thanks again^^
